src/haddock/libs/libsubprocess.py

```diff
@@ -45,20 +46,45 @@ def __init__(
             self,
             input_file,
             output_file,
-            cns_folder='.',
+            cns_folder,
+            modpath,
+            config_path,
             cns_exec=None,
             ):
         """
         CNS subprocess.
 
-        :param input_file: input CNS script
-        :param output_file: CNS output
-        :cns_folder: absolute execution path
-        :cns_exec: CNS binary including absolute path
+        Parameters
+        ----------
+        input_file : str or pathlib.Path
+            The path to the .inp CNS file.
+
+        output_file : str or pathlib.Path
+            The path to the .out CNS file, where the standard output
+            will be saved.
+
+        cns_folder : str of pathlib.Path
+            The path where the CNS scripts needed for the module reside.
+            For example, `modules/rigidibody/cns`.
+
+        mod_path : str of pathlib.Path
+            Path where the results of the haddock3 module executing this
+            CNS job will be saved.
+
+        config_path : str of pathlib.Path
```

Comment:
Here you mean the user configuration file, right?

Reply:
Yes, the path pointing to the user configuration file. But we need to folder, not the file itself. Let me check that.